src/Cache/QueryCache.php

```diff
@@ -26,6 +26,11 @@ public function __construct(CacheInterface $cache)
         $this->cache = $cache;
     }
 
+    public function clear(): bool
+    {
+        return $this->cache->clear();
```

Comment:
This is remove all cache in `CacheInterface` implementation, need remove only queries cache...